resources/definitions/fdmprinter.def.json

```diff
@@ -816,6 +816,15 @@
                     "minimum_value_warning": "0",
                     "maximum_value_warning": "machine_nozzle_size",
                     "settable_per_mesh": true
+                },"outer_inset_first": 
+                {
+                    "label": "Print Outer Inset First",
+                    "description": "Prints insets in order of outside to inside when selected. This can help improve dimensional accuracy in x and y when using a high viscosity plastic like ABS however it can decrease outer surface print quality, especially overhangs.",
```

Comment:
`when selected` should be `when enabled`
`X` and `Y` should always be capitalized in Cura.
There should be a semicolon `;` before `however`
`especially overhangs` ==> `especially on overhangs`